irohad/main/irohad.cpp

```diff
@@ -159,6 +172,62 @@ static shared_model::crypto::Keypair getKeypairFromFile(
              });
 }
 
+boost::optional<std::string> getEnvVar(std::string const& key)
+{
+    char const* val = getenv(key.c_str()); 
+    return val == NULL ? boost::optional<std::string>() : std::string(val);
```

Comment:
```suggestion
    return val == nullptr ? boost::optional<std::string>() : std::string(val);
```

Reply:
We highly recommend to use std:: alternatives instead of boost:: if they are.
```suggestion
    return val == nullptr ? std::nullopt : std::string(val);
```